mailbox/backup/src/test/java/org/apache/james/mailbox/backup/DefaultMailboxBackupTest.java

```diff
@@ -67,22 +63,22 @@
 
     private MailboxManager mailboxManager;
     private BlobStore store;
-    private File destination;
+    private ByteArrayOutputStream destination;
     private DefaultMailboxBackup backup;
 
     @BeforeEach
     void beforeEach(TemporaryFolderExtension.TemporaryFolder temporaryFolder) throws Exception {
-        destination = File.createTempFile("backup-test", ".zip", temporaryFolder.getTempDir());
+        int bufferSize = 4096;
+        destination = new ByteArrayOutputStream(bufferSize);
         mailboxManager = MemoryMailboxManagerProvider.provideMailboxManager(PRE_DELETION_HOOKS);
         store = new MemoryBlobStore(new HashBlobId.Factory());
         backup = new DefaultMailboxBackup(mailboxManager, archiveService, store);
     }
 
-    private void readFromStoreAndCopyInFile(BlobId blobId) throws Exception {
-        InputStream content = store.read(blobId);
-        try (OutputStream out = new FileOutputStream(destination)) {
-            IOUtils.copy(content, out);
-        }
+    private ZipFile getZipFileFromDestination() throws IOException {
```

Comment:
Should not be needed: rebase on master and use  `assertThatZip(destination)` directly

Reply:
Ok, since your PR i have just rebased. I prefer to have this method which enable me to close the zipfile and controle when i do it.
I tried to refactore assertThatZip(destination) to close the zipfile, but my test failed because the file was closed before the entry checks.

Reply:
```
try (ZipAssert zipAssert = assertThatZip(destination)) {
    zipAssert.containsEntries(...);
}
```

is not working? Do we have issues with this abstraction?

Reply:
i think i was not using it properly